CSharp/Library/Microsoft.Bot.Builder.Azure/AzureStorageQueueActivityLogger.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.History;
+using Microsoft.Bot.Connector;
+using Microsoft.ServiceBus.Messaging;
+using Microsoft.WindowsAzure.Storage.Queue;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Azure
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class AzureStorageQueueActivityLogger : IActivityLogger
+    {
+
+        private readonly JsonSerializerSettings _jsonSerializerSettings;
+        private CloudQueueClient _cloudQueueClient;
+        private CloudQueue _queueClient;
+        private QueueLoggerSettings _queueLoggerSettings;
+
+        private int _textMaxLength = 1024 * 20; //60k
+        private int _preCompressedMaxTextLength = 1024 * 50; //180k - assuming 3x compression
```

Comment:
I am getting very weird compression perf.  on 100K message, compression was only down to 81K.  Lets discuss, numbers above are def not final.